.pylintrc

```diff
@@ -443,7 +443,9 @@ disable=raw-checker-failed,
         attribute-defined-outside-init,
         abstract-method,
         pointless-statement,
-        wrong-import-order
+        wrong-import-order,
+        duplicate-code,
```

Comment:
Was there duplicate code in the launch_training that wasn't worth fixing? Seems like a good check to have instead of globally removing

Reply:
@anhuong What do you think?  I can revert this change and fix launch_training.py in another PR properly.
It looked a little messy because I had to put many` #pylint: disable=duplicate-code` in those 2 files. I could fix but then I have to figure out how to test it which could further delay this PR. 
```
build/launch_training.py:1:0: R0801: Similar lines in 2 files
==build.launch_training:[65:74]
==tuning.sft_trainer:[277:286]
    parser = transformers.HfArgumentParser(
        dataclass_types=(
            configs.ModelArguments,
            configs.DataArguments,
            configs.TrainingArguments,
            peft_config.LoraConfig,
            peft_config.PromptTuningConfig,
        )
    ) (duplicate-code)
build/launch_training.py:1:0: R0801: Similar lines in 2 files
==build.launch_training:[80:86]
==tuning.sft_trainer:[292:298]
        (
            model_args,
            data_args,
            training_args,
            lora_config,
            prompt_tuning_config, (duplicate-code)
```

Reply:
That is true I duplicated the code because passing the arguments to sft_trainer was causing issues and we wanted the same parsing of arguments. That's fine then, thanks for showing me what the output looks like